Code/Framework/AzToolsFramework/AzToolsFramework/Asset/AssetDebugInfo.cpp

```diff
@@ -198,10 +220,28 @@ namespace AzToolsFramework
             DependencyNode root;
             root.m_assetId = assetDebugInfo->second.m_assetId;
             BuildNodeTree(assetDebugInfo->second.m_assetId, &root);
-            root.BuildHumanReadableString("", *this);
+            root.BuildHumanReadableString("", *this, m_humanReadableString);
             m_humanReadableString += AZStd::string("\n");
         }
     }
+    void AssetFileDebugInfoList::BuildInvalidAssetHumanReadableString()
+    {
+        m_invalidAssetHumanReadableString = "\n\nThe following asset references do not exist:\n\n";
+        AZ_Error("AssetSeedManager", false, m_invalidAssetHumanReadableString.c_str());
+
+        for (auto const& assetId : m_invalidFileDebugInfoList)
+        {
+            auto assetDebugInfo = m_fileDebugInfoList.find(assetId);
+            if (assetDebugInfo != m_fileDebugInfoList.end())
+            {
+                DependencyNode root;
+                root.m_assetId = assetDebugInfo->second.m_assetId;
+                BuildNodeTree(assetDebugInfo->second.m_assetId, &root);
+                root.BuildHumanReadableString("", *this, m_invalidAssetHumanReadableString, true);
+                m_invalidAssetHumanReadableString += AZStd::string("\n");
```

Comment:
Instead of constructing an AZStd::string and appending it to the `m_invalidAssetHumanReadableString` and this can use the `char` overload of `basic_string::operator+=` to append a single character for minuscule benefits.
```suggestion
                m_invalidAssetHumanReadableString += '\n';
```